common/dynamicconfig/constants.go

```diff
@@ -917,6 +917,13 @@ const (
 	// Allowed filters: N/A
 
 	TimerTaskBatchSize
+	// TimerTaskDeleteBatchSize is batch size for timer processor to delete timer tasks
+	// KeyName: history.timerTaskDeleteBatchSize
+	// Value type: Int
+	// Default value: 4000
+	// Allowed filters: N/A
+
```

Comment:
nit: I guess the empty line should on L913 and 920 instead of 918 and 925?

nit: I am considering group the read batch size for all task types into one config and probably we can do the same for other configs as well. Not sure how useful it is to have separate configs.